extensions/dacpac/src/test/testDacFxService.ts

```diff
@@ -59,104 +59,109 @@ export class DacFxTestService implements mssql.IDacFxService {
-				verifyDeployment: { value: false, description: sampleDesc, displayName: sampleName },
-				ignoreComments: { value: false, description: sampleDesc, displayName: sampleName },
-				ignoreColumnCollation: { value: false, description: sampleDesc, displayName: sampleName },
-				ignoreAuthorizer: { value: false, description: sampleDesc, displayName: sampleName },
-				ignoreAnsiNulls: { value: false, description: sampleDesc, displayName: sampleName },
-				generateSmartDefaults: { value: false, description: sampleDesc, displayName: sampleName },
-				dropStatisticsNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-				dropRoleMembersNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-				dropPermissionsNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-				dropObjectsNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-				ignoreColumnOrder: { value: false, description: sampleDesc, displayName: sampleName },
-				doNotDropObjectTypes: { value: [], description: sampleDesc, displayName: sampleName },
-				excludeObjectTypes: { value: [], description: sampleDesc, displayName: sampleName },
-				ignoreTablePartitionOptions: { value: false, description: sampleDesc, displayName: sampleName },
-				doNotEvaluateSqlCmdVariables: { value: false, description: sampleDesc, displayName: sampleName },
-				disableParallelismForEnablingIndexes: { value: false, description: sampleDesc, displayName: sampleName },
-				disableIndexesForDataPhase: { value: false, description: sampleDesc, displayName: sampleName },
-				restoreSequenceCurrentValue: { value: false, description: sampleDesc, displayName: sampleName },
-				rebuildIndexesOfflineForDataPhase: { value: false, description: sampleDesc, displayName: sampleName },
-				isAlwaysEncryptedParameterizationEnabled: { value: false, description: sampleDesc, displayName: sampleName },
-				preserveIdentityLastValues: { value: false, description: sampleDesc, displayName: sampleName },
-				allowExternalLibraryPaths: { value: false, description: sampleDesc, displayName: sampleName },
-				allowExternalLanguagePaths: { value: false, description: sampleDesc, displayName: sampleName },
-				hashObjectNamesInLogs: { value: false, description: sampleDesc, displayName: sampleName },
-				doNotDropWorkloadClassifiers: { value: false, description: sampleDesc, displayName: sampleName },
-				ignoreWorkloadClassifiers: { value: false, description: sampleDesc, displayName: sampleName },
-				ignoreDatabaseWorkloadGroups: { value: false, description: sampleDesc, displayName: sampleName },
-				doNotDropDatabaseWorkloadGroups: { value: false, description: sampleDesc, displayName: sampleName }
+				ignoreTableOptions: { value: false, description: sampleDesc, propertyName: sampleName },
+				ignoreSemicolonBetweenStatements: { value: false, description: sampleDesc, propertyName: sampleName },
```

Comment:
do we need to have all the actual options here or can we just have a couple example test options like testOption1, testOption2 since they all have the same description and name anyway?

Reply:
These are not required anymore, as per our discussion way back, we need to work on these as a cleanup story. Do you want me to cover these changes with the current development and add a new PR for that?

Reply:
I'm not quite following. Can you clarify @ssreerama? In the test service here, why do we need another list of options? It would seem like we should have fewer sources of truth (and fewer places to update if more deployment options are added in the future)